cmd/cmrel/cmd/publish.go

```diff
@@ -40,10 +40,7 @@ Helm charts, generated static manifests and create a release tag on GitHub.
 
 It can only be run by specifying a previously staged build.
 `
-)
-
-var (
-	publishExample = fmt.Sprintf(``)
+	publishExample = ""
 )
```

Comment:
note: the diff is slightly unhelpful here; this changes publishExample into a constant